### ISSUE TITLE
Solutions to the questions?

### ISSUE THREAD

**Issue by gamename** [arush-sal/cka-practice-environment#9]
Hi,

I really enjoy the tool and think its a great way to study for the CKA.  

However, I would like to suggest you also supply the answers too.  For those of us studying for the cert, the answers may not be what we think.  It would be nice have a way to verify that. 

Thanks

**Comment by arush-sal** [comment]
@gamename That is a great suggestion. Let me work on that and will get that done sometime soon🤞

**Comment by Bougyz** [comment]
Hi, any luck with the solutions to the questions?

**Comment by kparashar** [comment]
Hello Arush, we are impatiently waiting for the solutions. Please try to post asap..

**Comment by moulishanmuhavelu** [comment]
Hi Arush,
Please upload the solutions when you get time.
In 4.html question, you mentioned to create a to mount PV. I hope just a deployment is enough for that. We don't need to create service for that. Please correct me if I'm wrong.

**Comment by akballow** [comment]
Need solutions too please, nothing worse then never knowing!

**Comment by ksingh7** [comment]
This question doesn't sounds right

![image](https://user-images.githubusercontent.com/9701902/72409874-fdba1000-378c-11ea-8fca-8a77b4af9c0b.png)

**Comment by ksingh7** [comment]
Instead of 2 pods it should be 2 containers

![image](https://user-images.githubusercontent.com/9701902/72411340-ac138480-3790-11ea-8997-d5c79c4a33b8.png)

**Comment by RadhaR4** [comment]
Do we have solutions to the questions?

**Comment by memark** [comment]
@arush-sal Did you find the time to upload the solutions?

**Comment by jpseara** [comment]
> Instead of 2 pods it should be 2 containers
> 
> ![image](https://user-images.githubusercontent.com/9701902/72411340-ac138480-3790-11ea-8997-d5c79c4a33b8.png)

No. It's correct this way. You have to use podAffinity for this answer.

**Comment by jpseara** [comment]
Hello, all.

I have created a project with all the answers and necessary yamls here [1].

Enjoy!

[1] https://github.com/jpseara/cka-practice-environment-answers

**Comment by himadriganguly** [comment]
For the first question "**Bootstrap a Kubernetes v1.10 cluster using kubeadm**" how we will get the master and the node machines.

**Comment by jpseara** [comment]
> For the first question "**Bootstrap a Kubernetes v1.10 cluster using kubeadm**" how we will get the master and the node machines.

In Minikube you can't. You can spin 3 VMs and test that.

**Comment by himadriganguly** [comment]
> > For the first question "**Bootstrap a Kubernetes v1.10 cluster using kubeadm**" how we will get the master and the node machines.
> 
> In Minikube you can't. You can spin 3 VMs and test that.

How can I do it from the exam terminal?

**Comment by jpseara** [comment]
> > > For the first question "**Bootstrap a Kubernetes v1.10 cluster using kubeadm**" how we will get the master and the node machines.
> > 
> > 
> > In Minikube you can't. You can spin 3 VMs and test that.
> 
> How can I do it from the exam terminal?

In the exam terminal, they give you access to all the machines you'll need.

**Comment by himadriganguly** [comment]
> > > > For the first question "**Bootstrap a Kubernetes v1.10 cluster using kubeadm**" how we will get the master and the node machines.
> > > 
> > > 
> > > In Minikube you can't. You can spin 3 VMs and test that.
> > 
> > 
> > How can I do it from the exam terminal?
> 
> In the exam terminal, they give you access to all the machines you'll need.

Mainly I want to know regarding this application context. On the exam, html page question is coming and the terminal so how can we do it from there? I have already created cluster in VirtualBox but then what is the utility of this application?

**Comment by jpseara** [comment]
> > > > > For the first question "**Bootstrap a Kubernetes v1.10 cluster using kubeadm**" how we will get the master and the node machines.
> > > > 
> > > > 
> > > > In Minikube you can't. You can spin 3 VMs and test that.
> > > 
> > > 
> > > How can I do it from the exam terminal?
> > 
> > 
> > In the exam terminal, they give you access to all the machines you'll need.
> 
> Mainly I want to know regarding this application context. On the exam, html page question is coming and the terminal so how can we do it from there? I have already created cluster in VirtualBox but then what is the utility of this application?

On the exam, they will give you ssh to the machines of the cluster you'll work in.
